utils/data_generator/datagen.py

```diff
@@ -46,6 +46,20 @@ def insert_data(token, kpi, value, municipality, year, dataseries=None):
     return json.loads(req.text)
 
 
+def insert_user(token, username, password, role):
```

Comment:
This should not be in the data generator, the purpose of which is to generate datapoints for the data entry api and goal distance computation. Should be part of the `make_user` utility.